README.md

```diff
@@ -75,12 +75,16 @@ a suite file defines a series of tests to run.
 #### suite file format
 
 ```ruby
+# format description:
 # each test can be executed by either target duration using :time => N secs
 # or by number of events with :events => N
 #
+# you can specify the number of filter worker threads for each test with :workers => N
+# if you don't specify workers, it defaults to 1
```

Comment:
I like more the idea of letting the tool user decide how many workers does he wants, then the expectations are more clear and should be less misunderstanding prone.